src/react/components/_academics/LawDegreeAudit/LawDegreeAudit.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+import { react2angular } from 'react2angular';
+import Widget from '../../Widget/Widget';
+import _ from 'lodash';
+import CampusSolutionsLinkContainer from '../../CampusSolutionsLink/CampusSolutionsLinkContainer';
```

Comment:
@nkutub we have been using a newer component called `<APILink />` to handle CS Link API links in React, in favor of `<CampusSolutionsLinkContainer />`.  Can you see if that is an option for you on this card?

Reply:
my component is not connected to redux, we should be able to continue to use the  CampusSolutionsLinkContainer since it does the job for simple components that are connected to the store.  personally I'm not a fan of the APILink implementation which requires it to be connected to redux in order to use.